CONTRIBUTING.md

```diff
@@ -0,0 +1,88 @@
+Contributing to WildFly Elytron
+==================================
+- [Forking the Project](#forking-the-project)
```

Comment:
Maybe we could add a sentence right before the table of contents that says:

Welcome to the WildFly Elytron project! We welcome contributions from the community. This guide will walk you through the steps for getting started on our project.

Reply:
Added

Reply:
Nice! What do you think about switching to regular text for this (instead of making it bold)?

Reply:
Oh yeah, it looked better on the vscode preview than here on github, I'll make it regular text